src/common/libflux/message.c

```diff
@@ -1446,16 +1446,14 @@ int flux_match_asprintf (struct flux_match *m, const char *fmt, ...)
 
```

Comment:
commit message typo "Modfiy" -> "Modify"

Reply:
Pushed that fix.